TTC.Deployment.AmazonWebServices/Deployer.cs

```diff
@@ -50,14 +57,16 @@ public class Deployer
                                                     stsCredentials.SecretAccessKey,
                                                     stsCredentials.SessionToken);
 
-                credentials = sessionCredentials;
+                awsConfiguration.Credentials = sessionCredentials;

```

Comment:
This is weird IMO. AWS configuration represents the command line arguments. Here, we update the object passed in to this method. I think this object should actually be immutable.